test/astar.cc

```diff
@@ -962,6 +962,92 @@ void test_time_restricted_road() {
   }
 }
 
+Api route_on_timerestricted(std::string& costing_str, int16_t hour) {
+  // Try routing over "Via Montebello" in Rome which is a time restricted road
+  // The restriction is
+  //
+  //    <tag k="auto" v="yes"/>
+  //    <tag k="motor_vehicle:conditional" v="no @ (Mo-Sa 07:00-16:00)"/>
+  //    <tag k="pedestrian" v="no"/>
+  //    <tag k="pedestrian:conditional" v="yes @ (Mo-Sa 08:00-15:00)"/>
```

Comment:
I had to tweak the tags manually in the osm-file in order to get the permutations of test-cases I needed.

Conditionals dropped here for clarity.